pihole

```diff
@@ -300,6 +300,29 @@ Branches:
   checkout "$@"
 }
 
+tricorderFunc() {
+  if [ ! -p "/dev/stdin" ]; then
+    echo "Please do not call Tricorder directly."
+    exit 1
+  fi
+
+  if command -v openssl &> /dev/null; then
```

Comment:
So I guess another request would be to add retcodes to the functions so they can be evaluated in other scripts.